src/App.js

```diff
@@ -14,9 +14,9 @@ export default class App extends Component {
     <Router>
       <header>
         <NavLink to="/" className="Link"> Home</NavLink>
-        <NavLink to="/update" className="Update"> Update Your Gaming library</NavLink>
-        <NavLink to='/create' className="Create"> Add a New Game to your library</NavLink>
-        <NavLink to='/list' className="List">View Your Gaming library</NavLink>
+        <NavLink to="/update" className="update"> Update Your Gaming library</NavLink>
```

Comment:
You don't need a link to the update page because you only want to access that page by clicking on a game so it can grab its id.